test/test_functest.py

```diff
@@ -51,8 +56,11 @@ def check_functest_sanitizers(implementation):
                  working_dir=os.path.join('..', 'test'),
                  env=env)
     helpers.run_subprocess(
-        ['./functest_{}_{}'.format(implementation.scheme.name,
-                                   implementation.name)],
+        [os.path.join('..', 'bin', 'functest_{}_{}{}'.format(
```

Comment:
Shouldn't need the ../bin added in the command, as that's been set as the working dir on line 64 below.